### ISSUE TITLE
Support for multimarkdown figure width

### ISSUE THREAD

**Issue by samoconnor** [mpastell/Weave.jl#34]
```diff
diff --git a/src/formatters.jl b/src/formatters.jl
index 2490246..46dddf0 100644
--- a/src/formatters.jl
+++ b/src/formatters.jl
@@ -285,20 +285,24 @@ end
 function formatfigures(chunk, docformat::Markdown)
     fignames = chunk.figures
     caption = chunk.options[:fig_cap]
+    width = "width=$(chunk.options[:out_width])"
     result = ""
     figstring = ""

     length(fignames) > 0 || (return "")

     if caption != nothing
-        result *= "![$caption]($(fignames[1]))\n"
+        result *= "![$caption][$(fignames[1])]\n\n"
+        result *= "[$(fignames[1])]: $(fignames[1]) $width\n"
         for fig = fignames[2:end]
-            result *= "![]($fig)\n"
+            result *= "![][$fig]\n\n"
+            result *= "[$fig]: $fig $width\n"
             println("Warning, only the first figure gets a caption\n")
         end
     else
         for fig in fignames
-            result *= "![]($fig)\n"
+            result *= "![][$fig]\n\n"
+            result *= "[$fig]: $fig $width\n"
         end
     end
     return result
```

**Comment by mpastell** [comment]
I think I will add multimarkdown as a separate output format, thanks for the suggestion.

**Comment by mpastell** [comment]
Do you know if  there other markdown variants that support this format? It seems Pandoc has an extension, but I plan to add Pandoc link_attributes to that formatter.

**Comment by samoconnor** [comment]
No, I'm not sure if its supported elsewhere.
At the time I evaluated pandoc and multimarkdown I liked multimardown's tables more. I haven't used anything other than multimarkdown + pdflatex since.

**Comment by mpastell** [comment]
What you see as reasonable default width for a figure? Other formats that support width have a default value for the chunks.

**Comment by mpastell** [comment]
I've added this to master, the format is called "multimarkdown", but it is still without default width so figures get the width=nothing, this doesn't seem to cause problems with HTML, but I'm not sure about other formats.

Could you test it?

**Comment by samoconnor** [comment]
I think the default should be to omit the width attribute.
Yes, I'll try it out, but I won't be able to get to it until Tuesday.

**Comment by abhijithch** [comment]
```
function foo(m,n)
    A =	sprand(m,n,0.1)
    for i = 1:n
for j =	i+1:n
            CosineMeasure = A[:,i]'*A[:,j]
        end
    end
end

@time foo(10000,1000)
```

**Comment by mpastell** [comment]
This is released with 0.1.1 with a small test case that works correctly so I'm going to close this issue. Let ne know if you run into any bugs.

**Comment by samoconnor** [comment]
Thanks Matti, I haven't had a chance to play with this yet, but I'll let you know when i do.
